discord_api.py

```diff
@@ -45,14 +44,11 @@
 		await channel.send(help(message))
 
 def checkAuthorization(author):
-	if author.name in authorized_authors:
+	if author.name in authorized_authors or author_restrict == False:
 		return True
-	else:
-		print(str(author.roles))
-		for i in author.roles:
-			print(str(i))
-			if i.name in authorized_roles:
-				return True
+	for i in author.roles:
+		if i.name in authorized_roles:
+			return True
 	return False
 
 def addFooter(message):
```

Comment:
Can we agree on a [coding style](https://www.python.org/dev/peps/pep-0008/) for some consistency?  Among other things, I see some function names in `camelCase`, while others are `snake_case`.

Reply:
I agree, so I've changed function and variable names in order to adapt code to the coding style